challenges/objects-arrays.js

```diff
@@ -48,22 +92,36 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
+
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+
+const universities = [
+  graduates.map(function(item){
```

Comment:
Would like to see you start to use arrow functions. And semantically it might make more sense to call your parameter here "graduate" since this is an array of objects, and each object represents a single graduate.
```suggestion
  graduates.map(graduate => {
```